src/Sentry.Unity.Native/SentryNative.cs

```diff
@@ -115,7 +111,7 @@ private static void ReinstallBackend()
         }
         catch (EntryPointNotFoundException e)
         {
-            Logger?.LogError(e, "Native dependency not found. Did you delete sentry.dll or move files around?");
+            Logger?.LogError("Native dependency not found. Did you delete sentry.dll or move files around?", e);
```

Comment:
Is this right? Usually last argument is `object[]` so it expects a `{0}` in the string.
The proper logging API has the exception as the first argument